v2/api-validator/tests/server-tests/collateral/account-link.test.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  CollateralAccount,
+  CollateralAccountLink,
+  CollateralLinkStatus,
+  CollateralSignerId,
+  CryptocurrencyReference,
+  Environment,
+} from '../../../src/client/generated';
+import { getCapableAccountId } from '../../utils/capable-accounts';
+import { Pageable, paginated } from '../../utils/pagination';
+import config from '../../../src/config';
+import Client from '../../../src/client';
+
+describe('Account Link', () => {
+  const client = new Client();
+
+  describe('Creates a new link between a collateral account and a provider account', () => {
+    describe.each([
+      { env: Environment.PROD, expectedTestAsset: false },
+      { env: Environment.SANDBOX, expectedTestAsset: true },
+    ])('Successful request', (testParams) => {
+      const { env, expectedTestAsset } = testParams;
+      it('Should return valid response', async () => {
+        const accountId: string = getCapableAccountId('collateral');
```

Comment:
What if there is no such account?

Reply:
updated all collateral test to use skipIf